//
// ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
// +                                                                      +
// + This file is part of enVisu.                                         +
// +                                                                      +
// + Copyright 2013 O. Gloth, enGits GmbH                                 +
// +                                                                      +
// + enGrid is free software: you can redistribute it and/or modify       +
// + it under the terms of the GNU General Public License as published by +
// + the Free Software Foundation, either version 3 of the License, or    +
// + (at your option) any later version.                                  +
// +                                                                      +
// + enGrid is distributed in the hope that it will be useful,            +
// + but WITHOUT ANY WARRANTY; without even the implied warranty of       +
// + MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the        +
// + GNU General Public License for more details.                         +
// +                                                                      +
// + You should have received a copy of the GNU General Public License    +
// + along with enVisu. If not, see <http://www.gnu.org/licenses/>.       +
// +                                                                      +
// ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
//

#ifndef NEWFILTERDIALOG_H
#define NEWFILTERDIALOG_H

#include <QDialog>

#include "ui_newfilterdialog.h"

class NewFilterDialog : public QDialog
{

  Q_OBJECT

  Ui::NewFilterDialog m_Ui;


private slots:

  void newInput();
  void editInput();
  void deleteInput();

public:

  explicit NewFilterDialog(QWidget *parent = 0);
  
signals:
  
public slots:
  
  void createFiles();

};

#endif // NEWFILTERDIALOG_H
